### ISSUE TITLE
About camera parameters

### ISSUE THREAD

**Issue by tszhang97** [facebookresearch/frankmocap#46]
Thanks for your amazing work. I am wondering how to use the predicted camera parameters to overlay the mesh. In SPIN, they use the camera parameters like this (focal=5000 and res=224):
camera_translation = torch.stack([pred_camera[:,1], pred_camera[:,2], 2*constants.FOCAL_LENGTH/(constants.IMG_RES * pred_camera[:,0] +1e-9)],dim=-1)

So, for frankmocap, can we transfer the predicted othog parameters to perspective camera parameters? Thanks a lot!

**Comment by penincillin** [comment]
Our camera parameters is similar to SPIN. 
For body and body-plus-hand, the camera parameters are exactly the same.
For hand, the camera parameters are also similar, but the order of applying translation and scaling are different.
BTW, I am wondering is there any difficulties here in using the provided code for overlay the mesh ?

**Comment by tszhang97** [comment]
Thanks for your reply! I am working on transferring the body model to c++ version and rewrite the rendering part. So far, it can achieve 25 fps on a 2080Ti (webcam input) but cannot overlay the output mesh.  I have tried using the same camera parameters of SPIN (focal length=5000 and img_res=224) and calculated the camera translation, but can not achieve the correct rendering results.  I will check it out.

**Comment by penincillin** [comment]
For projection, it is easy for you to use orthogonal projection in C++ code ? If it is, you can follow the similar way to get [pred_vertices_smpl](https://github.com/facebookresearch/frankmocap/blob/f1a85d09e4e84b61a200d9725a5c811d09fd033e/bodymocap/body_mocap_api.py#L110) and apply orthogonal rendering to get the overlay mesh.
